Frameworks/UIKit/UIWindow.mm

```diff
@@ -311,6 +255,21 @@ - (void)setRootViewController:(UIViewController*)controller {
     }
 }
 
+/**
+ @Status Interoperable
+*/
+- (BOOL)sizeUIWindowToFit {
+    return _sizeUIWindowToFit;
```

Comment:
>_sizeUIWindowToFit [](start = 11, length = 18)

i could not find sizeUIWindowToFit on ref doc. and it seems we only cache a value and did not do anything using it anyway?